MantidQt/CustomInterfaces/src/Indirect/IndirectTransmission.cpp

```diff
@@ -147,7 +148,8 @@ void IndirectTransmission::saveClicked() {
  * Handle mantid plotting
  */
 void IndirectTransmission::plotClicked() {
-  QString outputWs = (m_uiForm.dsSampleInput->getCurrentDataName() + "_trans");
+  QString outputWs =
+      (m_uiForm.dsSampleInput->getCurrentDataName() + "_transmission");
```

Comment:
Would be good to refactor this so the string `_transmission` only appears in one place, e.g. as a constant. (Outside scope of this issue, can be done as a future maintenance task)